src/styles/partials/components/_site-header.scss

```diff
@@ -70,6 +84,13 @@
       padding: 0;
     }
   }
+
+  #search-form {
+    .dg_search-btn {
+      right: 30px;
+      top: 20px;
+    }
+  }
 }
 
 $small-mobile-button-size: 65px;
```

Comment:
youre search form uses 65px as well below. Could this be named something more generic to accommodate that as well?

 #search-form {
    top: 65px;
  }

Reply:
good catch, yea, i think its just header-height. Will update.

Reply:
jsut relaized these shouldn't be showing up in this pr.